Tests/SwiftSyntaxBuilderTest/SourceFileTests.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+final class SourceFileTests: XCTestCase {
+  func testSourceFile() {
+    let source = SourceFile(eofToken: .eof, statementsBuilder: {
```

Comment:
I think it would be good to hand-craft a 
```swift
public init(@CodeBlockItemListBuilder statements: () -> ExpressibleAsCodeBlockItemList)
```
initializer for `SourceFile`. That way this line could be
```suggestion
    let source = SourceFile {
```